src/app.ts

```diff
@@ -1,8 +1,15 @@
 import { Sequelize } from "sequelize";
 
-import express = require("express");
-import cors = require("cors");
-import logger = require("morgan");
+const express = require("express");
+const cors = require("cors");
+const logger = require("morgan");
+const cookieParser = require('cookie-parser');
+const rateLimit = require('express-rate-limit');
+const helmet = require('helmet');
+const xss = require('xss-clean');
+const hpp = require('hpp');
+const cors = require('cors');

```

Comment:
Hey @Braineanear, thanks for this PR. Please fix the duplicated imports and resend.